cmd/backfill/main.go

```diff
@@ -0,0 +1,274 @@
+package main
+
+import (
+	"bytes"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/guregu/null"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+const (
+	envFile     = "BACKFILL_FILE"
+	envHost     = "BACKFILL_HOST"
+	envAuth     = "BACKFILL_AUTH"
+	healthcheck = "https://%s/api/v1/healthcheck"
+)
+
+type config struct {
+	file string
+	host string
+	auth string
+}
+
+func main() {
+	cfg := &config{
+		file: os.Getenv(envFile),
+		host: os.Getenv(envHost),
+		auth: os.Getenv(envAuth),
+	}
+	if err := execute(cfg); err != nil {
+		fmt.Fprintf(os.Stdout, "backfill failure: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func execute(cfg *config) error {
+	url := fmt.Sprintf(healthcheck, cfg.host)
+
+	/* #nosec G107 */
+	if _, err := http.Get(url); err != nil {
+		return fmt.Errorf("failed healthcheck for [%s]: %w", url, err)
+	}
+
+	f, err := os.Open(cfg.file)
+	if err != nil {
+		return err
+	}
+	src := csv.NewReader(f)
+	row, err := src.Read()
+	if err != nil {
+		return fmt.Errorf("failed to read column headers for [%s]: %w", cfg.file, err)
+	}
+	// fmt.Fprintf(os.Stdout, "headers: %v\n", row)
+	_ = row
```

Comment:
Just curious about this assignment with the blank identifier here. I'm assuming it was just there for debugging before you actually used `row`?

Reply:
That was to shut up the linter, for an "ineffectual assignment"... I was counting on this line as the definition of `row` so that I could use `row, err = ...` (without colon) below in the loop.